StudyBuddy/Abstractions/IUserManager.cs

```diff
@@ -10,4 +10,5 @@ public interface IUserManager
     IUser? GetUserByUsername(string username);
     void BanUser(UserId userId);
     List<IUser> GetAllUsers();
+    public void LoadUsersFromCsv(string filePath);
```

Comment:
Very bad practise to have access modifiers in interface.

Reply:
Good spot